libopenage/versions/versions.cpp

```diff
@@ -1,36 +1,71 @@
 // Copyright 2020-2020 the openage authors. See copying.md for legal info.
 
-#include <gnu/libc-version.h>
+#include "versions.h"
+
+#ifndef __APPLE__
+  #include <gnu/libc-version.h>
+  #define TARGET_NOT_OSX
```

Comment:
this new preprocessor var is just another indirection and doesn't make it easier for the brain, thus we should just use `__APPLE__` later in the file